lib/android/src/main/java/net/skyscanner/backpack/reactnative/rating/BpkRatingViewManager.kt

```diff
@@ -117,10 +117,11 @@ class BpkRatingViewManager : BaseViewManager<RNBpkRating, BpkRatingShadowNode>()
     }
   }
 
-  private fun String.asRatingOrientation(): BpkRating.Orientation {
+  private fun String.asRatingStyle(): BpkRating.Style {
     return when (this.toLowerCase(Locale.ROOT)) {
-      "horizontal" -> BpkRating.Orientation.Horizontal
-      "vertical" -> BpkRating.Orientation.Vertical
+      "pill" -> BpkRating.Style.Horizontal
```

Comment:
Do we need to update the readme to say that `pill` is supported?
https://backpack.github.io/components/rating/?platform=native#implementation

Reply:
Plus one to this!